addons/website_event_track/models/website.py

```diff
@@ -34,10 +34,10 @@ def _compute_app_icon(self):
             App Icon should be in PNG format and size of at least 512x512.
         """
         for website in self:
-            if not website.favicon:
+            image = ImageProcess(website.favicon) if website.favicon else None
+            if not (image and image.image):
```

Comment:
This 100% is going to be re-introduced at some point if we don't say anything.

Test would be great, but there should at least be a comment for SVGs